ACRA getDebugInfo: android.util.AndroidRuntimeException: android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed

https://ankidroid.org/acra/app/1/bug/314673/report/69fc797c-f611-42a8-88b1-f064ac3173c2

```
android.util.AndroidRuntimeException: android.util.AndroidRuntimeException: android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed
	at android.webkit.WebViewFactory.getProvider(WebViewFactory.java:363)
	at android.webkit.WebView.getFactory(WebView.java:2605)
	at android.webkit.WebView.ensureProviderCreated(WebView.java:2599)
	at android.webkit.WebView.setOverScrollMode(WebView.java:2667)
	at android.view.View.<init>(View.java:5522)
	at android.view.View.<init>(View.java:5663)
	at android.view.ViewGroup.<init>(ViewGroup.java:704)
	at android.widget.AbsoluteLayout.<init>(AbsoluteLayout.java:56)
	at android.webkit.WebView.<init>(WebView.java:421)
	at android.webkit.WebView.<init>(WebView.java:363)
	at android.webkit.WebView.<init>(WebView.java:345)
	at android.webkit.WebView.<init>(WebView.java:332)
	at android.webkit.WebView.<init>(WebView.java:322)
	at com.ichi2.utils.WebViewUtilsKt.getWebviewUserAgent(WebViewUtils.kt:57)
	at com.ichi2.anki.servicelayer.DebugInfoService$getDebugInfo$webviewUserAgent$1.invokeSuspend(DebugInfoService.kt:39)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at android.os.Handler.handleCallback(Handler.java:1000)
	at android.os.Handler.dispatchMessage(Handler.java:104)
	at android.os.Looper.loopOnce(Looper.java:242)
	at android.os.Looper.loop(Looper.java:362)
	at android.app.ActivityThread.main(ActivityThread.java:8448)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:992)
Caused by: android.util.AndroidRuntimeException: android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed
	at android.webkit.WebViewFactory.getProviderClass(WebViewFactory.java:532)
	at android.webkit.WebViewFactory.getProvider(WebViewFactory.java:348)
	... 24 more
Caused by: android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed
	at android.webkit.WebViewFactory.getWebViewContextAndSetProvider(WebViewFactory.java:428)
	at android.webkit.WebViewFactory.getProviderClass(WebViewFactory.java:493)
	... 25 more
```

--- a/AnkiDroid/src/main/java/com/ichi2/utils/WebViewUtils.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/utils/WebViewUtils.kt
@@ -19,6 +19,7 @@ package com.ichi2.utils
 import android.content.Context
 import android.content.pm.PackageInfo
 import android.content.pm.PackageManager
+import android.util.AndroidRuntimeException
 import android.webkit.WebView
 import androidx.annotation.MainThread
 import androidx.annotation.StringRes
@@ -56,6 +57,13 @@ fun checkWebviewVersion(activity: AnkiActivity) {
 fun getWebviewUserAgent(context: Context): String? {
     try {
         return WebView(context).settings.userAgentString
+    } catch (e: AndroidRuntimeException) {
+        // MissingWebViewPackageException is not public
+        if (e.cause.toString().contains("MissingWebViewPackageException")) {
+            Timber.w(e, "MissingWebViewPackageException")
+            return null // WebView not installed - don't log a crash report
+        }
+        CrashReportService.sendExceptionReport(e, "WebViewUtils", "some issue occurred while extracting webview user agent")
     } catch (e: Throwable) {
         CrashReportService.sendExceptionReport(e, "WebViewUtils", "some issue occurred while extracting webview user agent")
     }

